google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPoolOptions.java

```diff
@@ -180,6 +190,18 @@ public boolean isAutoDetectDialect() {
     return autoDetectDialect;
   }
 
+  public boolean closeInactiveTransactions() {
```

Comment:
There are two conventions being followed across languages. My guess is, the external options (within `SpannerOptions` in case of Java) has references to `InactiveTransaction` since that's how we are explaining the feature to to an external customer. Any other files can use 'long-running sessions' terminology.